src/CoreBundle/Assets/IconBuilder.php

```diff
@@ -25,6 +26,7 @@
 use Contao\CoreBundle\Image\ImageFactoryInterface;
 use Contao\Validator;
 use Symfony\Component\Filesystem\Filesystem;
+use Webmozart\PathUtil\Path;
```

Comment:
Is this a hidden dependency that needs to be added to the composer.json manifest?

Reply:
No, it's already a dependency: 

https://github.com/MetaModels/core/blob/a85c97cc492faf691665b6c994e79642bd6b2c15/composer.json#L51